.github/workflows/create_cluster.yml

```diff
@@ -101,6 +107,9 @@ jobs:
           terraform workspace delete -force ${{ matrix.destroyable_resources }}-infrastructure
           terraform workspace delete -force ${{ matrix.destroyable_resources }}-cluster
 
+      - name: Remove kubeconfig
```

Comment:
Why do we assume that runner will always have these files? The runner may be recreated.